block-buffer/src/lib.rs

```diff
@@ -40,6 +40,19 @@ pub type EagerBuffer<B> = BlockBuffer<B, Eager>;
 /// Lazy block buffer.
 pub type LazyBuffer<B> = BlockBuffer<B, Lazy>;
 
+/// Block buffer error.
+#[derive(Copy, Clone, Eq, PartialEq, Debug)]
+pub struct Error;
+
+impl fmt::Display for Error {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> Result<(), fmt::Error> {
+        f.write_str("Block buffer error")
+    }
+}
+
+#[cfg(feature = "std")]
+impl std::error::Error for Error {}
```

Comment:
`block-buffer` does not have `std` feature. Since the `try_new` method is intended to be used only by our crates with immediate remap into a different error, I think we can omit implementation of the `Error` trait.